pyeudiw/tests/oauth2/test_dpop.py

```diff
@@ -65,18 +65,40 @@ def wia_jws(jwshelper):
 
 def test_create_validate_dpop_http_headers(wia_jws, private_jwk=PRIVATE_JWK):
     # create
-    unpad_jwt_header(wia_jws)
+    header = unpad_jwt_header(wia_jws)
+    assert header
+    assert header["trust_chain"] == []
+    assert header["x5c"] == []
+    assert header["alg"]
+    assert header["kid"]
+    
     payload = unpad_jwt_payload(wia_jws)
```

Comment:
this is a test for the Wallet Instance Attestation while in this unit test we aims to test the DPoP proof

since you have did this test for the WIA, please move it to another test, and please in this unit test just test the DPoP proof, public key and ath binding